<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anoname</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Race</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Combo_Race">
          <item>
           <property name="text">
            <string>Nain</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Elfe</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Humain</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Orque</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gnome</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Halfelin</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Sexe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Combo_Gender">
          <property name="currentText">
           <string>Homme</string>
          </property>
          <item>
           <property name="text">
            <string>Homme</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Femme</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Txt_Name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Btn_Set">
          <property name="text">
           <string>Générer .set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_Name">
          <property name="text">
           <string>Générer nom</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>182</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
